src/main/java/application/model/User.java

```diff
@@ -22,7 +26,12 @@ public User(Long id, String name, String login, String password) {
         this.password = password;
     }
 
-    public User() {
+    public User(Long id, String name, String login, String password, byte[] salt) {
```

Comment:
so many constructors, do we use them all?

Reply:
done